include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -448,42 +451,64 @@ TridiagResult<T, Device::CPU> BandToTridiag<Backend::MC, D, T>::call_L(
       dlaf::internal::whenAllLift(start, std::min(nb, size - start), std::min(nb, size - 1 - start),
                                   mat_trid.readwrite_sender(GlobalTileIndex{0, tile_index}),
                                   std::forward<decltype(dep)>(dep)) |
-          dlaf::internal::transformDetach(policy_hp, copy_tridiag_task);
+          dlaf::internal::transformDetach(policy, copy_tridiag_task);
+    }
+    else {
+      ex::start_detached(std::forward<decltype(dep)>(dep));
```

Comment:
Not all input senders need to be started (and in that case this is an unnecessary allocation). Does that ever happen?